lib/host.rb

```diff
@@ -511,31 +511,24 @@ def delete(file, opts={})
         if bad_files.include? file
           raise "should not remove file named '#{file}'"
         end
-
-        r = "-r"
+        meth = :rm
```

Comment:
It looks like `rm_r` is recursive

```ruby
        meth = :rm_r
```

Reply:
or it looks like there is a method already

https://rubydoc.info/stdlib/fileutils/FileUtils#remove_entry_secure-class_method

This method removes a file system entry path. path shall be a regular file, a directory, or something. If path is a directory, remove it recursively. This method is required to avoid TOCTTOU (time-of-check-to-time-of-use) local security vulnerability of rm_r. #rm_r causes security hole when:

Parent directory is world writable (including /tmp).

Removing directory tree includes world writable directory.

The system has symbolic link.